app/routes/languages.rb

```diff
@@ -5,21 +5,18 @@ module Routes
     class Languages < Core
       get '/languages' do
         tracks = X::Track.all
+        tracks.delete_if { |t| t.repository == 'https://github.com/exercism/xSLUG' }
```

Comment:
Let me fix this in the data, rather than munging it here. Shouldn't take but a moment.